VirtusLab/strapi-plugin-comments
# feat: strapi 3.1.0 rbac support

### What has been done?

Added support for strapi 3.1.0 features including RBAC.

### How to test it?

1. Login as an admin
2. See that you're able to use all the features
3. Go to Settings -> Roles
4. Select Editor or Reader
5. Go to Plugins -> Comments and add `read` permission
6. Login as Editor or Reader
7. See that you're able only to see the plugin, no moderation actions are allowed.
